.rustfmt.toml

```diff
@@ -0,0 +1,4 @@
+# Doc: https://rust-lang.github.io/rustfmt/
+imports_granularity="Crate"
+group_imports="StdExternalCrate"
+hex_literal_case="Lower"
```

Comment:
In additional to these, I think `wrap_comments = true` is also helpful

Reply:
Added